src/app/pages/back-to-office/back-to-office.page.html

```diff
@@ -1,10 +1,22 @@
 <ion-header>
-  <ion-toolbar>
-    <ion-title>back-to-office</ion-title>
-  </ion-toolbar>
+    <practice-mode-banner *ngIf="isEndToEndPracticeMode"></practice-mode-banner>
 </ion-header>
 
-<ion-content>
-  <ion-button (click)="navigateBack()">Back</ion-button>
-  <ion-button (click)="navigateForward()">To Office</ion-button>
+<ion-content id="post-debrief-holding-cat-b-page" class="background-colour">
+    <div class="container">
+        <img id='dvsa-logo' src='../../../assets/imgs/back-to-office/logo.png' />
+    </div>
+    <div class="container">
+        <ion-button id="continue-to-write-up"
+                    class="mes-neutral-button office-button top-button"
+                    (click)="navigateBack()">Continue to write up
```

Comment:
```navigateBack()``` seems misleading here as this button would typically navigate you forwards to the office page. See here for example https://github.com/dvsa/mes-mobile-app/blob/develop/src/pages/back-to-office/cat-b/back-to-office.cat-b.page.html